device/mellanox/x86_64-nvidia_sn5400-r0/ACS-SN5400/hwsku.json

```diff
@@ -0,0 +1,202 @@
+        },
+        "Ethernet304": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet312": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet320": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet328": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet336": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet344": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet352": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet360": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet368": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet376": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet384": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet392": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet400": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet408": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet416": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet424": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet432": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet440": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet448": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet456": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet464": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet472": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet480": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet488": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet496": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet504": {
+            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G]"
+        },
+        "Ethernet512": {
+            "default_brkout_mode": "1x25G[10G]"
+        },
+        "Ethernet513": {
```

Comment:
Need to check if the fact this port is still mentioned here will not make a problem